csharp/ql/src/Bad Practices/PathCombine.ql

```diff
@@ -15,4 +16,4 @@ import semmle.code.csharp.frameworks.System
 
 from MethodCall call
 where call.getTarget().hasFullyQualifiedName("System.IO", "Path", "Combine")
-select call, "Call to 'System.IO.Path.Combine'."
+select call, "Call to 'System.IO.Path.Combine' may silently drop its earlier arguments."
```

Comment:
A test case is failing due to the change in the alert message. The `.expected` file needs to be updated.
The test expected output can be updated by running
```
codeql test run <path_to_test> --learn
```
Alternatively, the `.expected` file can be edited by hand.